pkg/apis/servicecatalog/validation/serviceplan.go

```diff
@@ -26,49 +27,47 @@ import (
 	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
-const servicePlanNameFmt string = `[-.a-zA-Z0-9]+`
-const servicePlanNameMaxLength int = 63
+const commonServicePlanNameFmt string = `[-.a-zA-Z0-9]+`
+const commonServicePlanNameMaxLength int = 63
 
-var servicePlanNameRegexp = regexp.MustCompile("^" + servicePlanNameFmt + "$")
+var servicePlanNameRegexp = regexp.MustCompile("^" + commonServicePlanNameFmt + "$")
 
-// validateServicePlanName is the validation function for ServicePlan names.
-func validateServicePlanName(value string, prefix bool) []string {
+// validateCommonServicePlanName is the common validation function for
+// service plan types.
+func validateCommonServicePlanName(value string, prefix bool) []string {
 	var errs []string
-	if len(value) > servicePlanNameMaxLength {
-		errs = append(errs, utilvalidation.MaxLenError(servicePlanNameMaxLength))
+	if len(value) > commonServicePlanNameMaxLength {
+		errs = append(errs, utilvalidation.MaxLenError(commonServicePlanNameMaxLength))
 	}
 	if !servicePlanNameRegexp.MatchString(value) {
-		errs = append(errs, utilvalidation.RegexError(servicePlanNameFmt, "plan-name-40d-0983-1b89"))
+		errs = append(errs, utilvalidation.RegexError(commonServicePlanNameFmt, "plan-name-40d-0983-1b89"))
 	}
 
 	return errs
 }
 
 // ValidateClusterServicePlan validates a ClusterServicePlan and returns a list of errors.
 func ValidateClusterServicePlan(serviceplan *sc.ClusterServicePlan) field.ErrorList {
```

Comment:
If you are trying to rename ALL THE THINGS, you missed the param for this function, it's still `serviceplan` and not `clusterServicePlan`.

Reply:
oooh thanks!